app/src/pages/playground/fetchPlaygroundPrompt.ts

```diff
@@ -158,11 +164,22 @@ export const promptVersionToInstance = ({
       __type: "chat",
       messages:
         "messages" in promptVersion.template
-          ? promptVersion.template.messages.map((m) => ({
-              ...m,
-              id: generateMessageId(),
-              role: getChatRole(m.role?.toLocaleLowerCase() as string),
-            }))
+          ? promptVersion.template.messages.map((m) => {
```

Comment:
nit: name the variable so you can read the code block at any point of the inner function block
```suggestion
          ? promptVersion.template.messages.map((message) => {
```